utils/api.py

```diff
@@ -459,11 +459,15 @@ def __init__(self, obj):
 
     def reload(self):
         self._obj.reload_if_needed()
+        reloaded_actions = []
         for action in self._obj._actions:
-            setattr(
-                self,
-                action["name"],
-                Action(self, action["name"], action["method"], action["href"]))
+            # don't redefine actions with same name and different methods
```

Comment:
If there are two with teh same name, how do we know we're picking the right one?